src/Exceptions/Handler.php

```diff
@@ -116,28 +117,14 @@ public function render($request, Exception $e)
 
         $this->isJsonOutputFormat = $request->ajax() || $request->wantsJson();
 
-        /*
-         * See Laravel 5.4 Changelog https://laravel.com/docs/5.4/upgrade
-         * The Illuminate\Http\Exception\HttpResponseException has been renamed to Illuminate\Http\Exceptions\HttpResponseException.
-         * Note that Exceptions is now plural.
-         */
-        $laravel53HttpResponseException = 'Illuminate\Http\Exception\HttpResponseException';
-        $laravel54HttpResponseException = 'Illuminate\Http\Exceptions\HttpResponseException';
-
-        $httpResponseExceptionClass = class_exists($laravel54HttpResponseException) ? $laravel54HttpResponseException : $laravel53HttpResponseException;
-
-        if ($e instanceof $httpResponseExceptionClass) {
+        if ($e instanceof HttpResponseException) {
             return $e->getResponse();
         } elseif ($e instanceof AuthenticationException) {
             return $this->handleUnauthenticated($request, $e);
         } elseif ($e instanceof ValidationException) {
             return $this->convertValidationExceptionToResponse($e, $request);
         }
 
-        if ($this->config->get('app.debug', false) && $this->config->get('twill.debug.use_inspector', false)) {
-            return Inspector::renderException($e);
-        }
-
         if ($this->config->get('app.debug', false) && $this->config->get('twill.debug.use_whoops', false)) {
```

Comment:
The `twill.debug.use_whoops` configuration key and other Whoops relation keys can be removed as well which means we can remove this and the `renderExceptionWithWhoops` method.

Reply:
ok, done.